src/platform.ts

```diff
@@ -213,9 +213,9 @@ export async function closeFdAsync(fd: number): Promise<void> {
   return await promisify(nodeFS.close)(fd);
 }
 
-export function getMimeType(file: string): string | null {
-  assertFileAccess();
-  return mime.getType(file);
+export function getMimeType(file: string): string {
+  const extension = file.substring(file.lastIndexOf('.') + 1);
+  return extensionToMime[extension] || 'application/octet-stream';
```

Comment:
Please remove `import * as mime from 'mime'` above, and 'mime' entry from `src/web.webpack.config.js`.